.github/workflows/build.yml

```diff
@@ -1,5 +1,8 @@
 name: Build Slinky docker images
 on:
+  create:
+    tags:
+      - "v.*"
```

Comment:
Should this be `v*`?

Reply:
i think youre right, i dont think we even need this create thing lol